tox.ini

```diff
@@ -60,15 +48,23 @@ basepython =
     py310: python3.10
     py311: python3.11
 
-[testenv:lint]
-basepython=python
+[common-lint]
 extras=linter
 commands=
     flake8 {toxinidir}/web3 {toxinidir}/ens {toxinidir}/ethpm {toxinidir}/tests --exclude {toxinidir}/ethpm/ethpm-spec,{toxinidir}/**/*_pb2.py
     black {toxinidir}/ens {toxinidir}/ethpm {toxinidir}/web3 {toxinidir}/tests {toxinidir}/setup.py --exclude /ethpm/ethpm-spec/|/ethpm/_utils/protobuf/ipfs_file_pb2\.py --check
-    isort --recursive --skip {toxinidir}/ethpm/_utils/protobuf/ipfs_file_pb2.py --skip {toxinidir}/ethpm/ethpm-spec --check-only --diff {toxinidir}/web3/ {toxinidir}/ens/ {toxinidir}/ethpm/ {toxinidir}/tests/
+    isort --check-only --diff {toxinidir}/web3/ {toxinidir}/ens/ {toxinidir}/ethpm/ {toxinidir}/tests/
     mypy -p web3 -p ethpm -p ens --config-file {toxinidir}/mypy.ini
 
+[testenv:lint]
```

Comment:
Do we still need this here?

Reply:
Yeah this allows for `make lint` locally since the Makefile uses `tox -e lint`. We need that generic "lint" tox env that uses whatever python your interpreter is using for linting (`basepython = python`).